notifications_utils/recipients.py

```diff
@@ -230,8 +233,12 @@ def get_rows(self):
                 yield None
 
     @property
-    def more_rows_than_can_send(self):
```

Comment:
do you need to keep this for backward compatibility? Its used in the API right? this way if the old code exists - it will work until we make the changes with the new funcs

Reply:
can we keep the old code? this way there is no requirement on merge order

Reply:
Done!